src/lib/error-handler/index.ts

```diff
@@ -61,7 +61,7 @@ export const errorHandler = function errorHandler(
 ): ExResponse | void {
   if (err instanceof ValidateError) {
     logger.warn(`Handled Validation Error for ${req.path}: %s`, JSON.stringify(err.fields))
-    const { status, ...rest } = err
+    const { ...rest } = err
```

Comment:
this change is causing the attachment test to fail

Reply:
Yeah, I've taken that out

Reply:
the change is still there?

Reply:
updated, please validate when you get a chance.